packages/ag-charts-community/benchmarks/timing.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
```

Comment:
optional: consider using `eslint-disable-next-line` because the console is only referenced once in this file

Reply:
Done